port_ocean/core/handlers/webhook/webhook_event.py

```diff
@@ -75,3 +77,22 @@ def set_timestamp(self, timestamp: WebhookEventTimestamp) -> None:
                 "timestamp_type": timestamp.value,
             },
         )
+        self._timestamp = timestamp
+
+
+class WebhookEventData:
+    """
+    Class for webhook event to store the updated data for the event
+    """
+
+    def __init__(self, kind: str, data: list[RAW_ITEM]) -> None:
+        self._kind = kind
+        self._data = data
+
+    @property
+    def kind(self) -> str:
+        return self._kind
+
+    @property
+    def data(self) -> list[RAW_ITEM]:
+        return self._data
```

Comment:
why kind is required to be returned, you already know the context of the kind that triggered the handler matched on